### ISSUE TITLE
save the deck after going idle for 60 seconds

### ISSUE THREAD

**Issue by SirZach** [SirZach/bbbbbbbbbbbbbbb#32]
it could be a nice feature to have autosave when you're creating a deck but this might be overkill to the firebase data flow.

@ahaurw01 thoughts?

**Comment by ahaurw01** [comment]
Sounds like a nice idea. Probably would prioritize low for me personally.

**Comment by SirZach** [comment]
agreed. i was just concerned last night when i was making a deck that nothing was saving because there wasn't any indication that something was happening

**Comment by ahaurw01** [comment]
Ah yeah. I definitely like this idea. It might have to be given a temporary name of some sort, but totally doable. The toasts will definitely help otherwise.
